### PR TITLE
[8.0][FIX] Workflow button generate_project

### PR DESCRIPTION
**Pull request by Omarius1981**
[Fix]  #35  Fix workflow button generate_project

### CODE REVIEW

File: business_requirement_deliverable_project/views/business_view.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
                             type="object"
                             class="oe_highlight"
                             groups="business_requirement.group_business_requirement_manager"
-                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'approved')]}"/>
+                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'stakeholder_approval')]}"/>
```
Comment (elicoidal) [inline business_requirement_deliverable_project/views/business_view.xml]: @Omarius1981 should be "stakeholder_approved"

**Review by elicoidal — CHANGES_REQUESTED** [review]

**Review by andhit-r — APPROVED** [review]
:+1:

File: business_requirement_deliverable_project/views/business_view.xml (hunk 1 above)
Comment (elicoidal) [reply business_requirement_deliverable_project/views/business_view.xml]: @Omarius1981 any news on this change request?

Comment (astirpe) [reply business_requirement_deliverable_project/views/business_view.xml]: @elicoidal if we apply the change of the name as in https://github.com/sudhir-serpentcs/business-requirement/pull/9 also to this module for V8, then the fix above would be fine, right?

**Comment by elicoidal** [comment]
agree

**Comment by rafaelbn** [comment]
Hi @Omarius1981 "This branch has conflicts that must be resolved", could you fix this?
Please also review @elicoidal comments, thanks!

**Review by susport — COMMENTED** [review]
👍

File: business_requirement_deliverable_project/views/business_view.xml (hunk 1 above)
Comment (elicoidal) [reply business_requirement_deliverable_project/views/business_view.xml]: @astirpe correct.
I am back now and I am trying to check the project.
What is the status of this PR?
Runbot is down but I just forced a rebuild now

**Comment by elicoidal** [comment]
@Omarius1981 Any news?
I would like to move forward on this PR now: let me know if you need some help.
Can you rebase so that we take up from there at least?

File: business_requirement_deliverable_project/views/business_view.xml (hunk 1 above)
Comment (andhit-r) [reply business_requirement_deliverable_project/views/business_view.xml]: @elicoidal based on https://github.com/OCA/business-requirement/blob/8.0/business_requirement/models/business.py#L218, @Omarius1981 fix is correct.

**Comment by elicoidal** [comment]
@andhit-r not sure whether I resolved properly the conflicts but do agree with the change
 




------------------
--
 	    Eric Caudal [Founder and CEO]
                              Skype: elico.corp. Phone: + 86 186 2136 1670 (Cell), + 86 21 6211 8017/27/37 (Office)             
             Elico Shanghai (Hong Kong/Shenzhen/Singapore)             
Odoo Gold Partner // Best Odoo Partner APAC 2014 and 2016
 

 
 
 
------------------ Original ------------------
From: "Andhitia Rama"; 
Date: 2017年2月24日(星期五) 下午3:52
To: "OCA/business-requirement"; 
Cc: "Eric @ Elico Corp"; "Mention"; 
Subject: Re: [OCA/business-requirement] 8.0 Fix workflow buttongenerate_project (#38)

 

@andhit-r commented on this pull request.
 
 
In business_requirement_deliverable_project/views/business_view.xml:
 > @@ -41,7 +41,7 @@                              type="object"                              class="oe_highlight"                              groups="business_requirement.group_business_requirement_manager" -                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'approved')]}"/> +                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'stakeholder_approval')]}"/>  
@elicoidal based on https://github.com/OCA/business-requirement/blob/8.0/business_requirement/models/business.py#L218, @Omarius1981 fix is correct.

**Review by elicoidal — APPROVED** [review]

**Pull request merged**